contribs/examples/enabled-video.sh

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env bash
+# Copyright 2022 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+set -e
+set -u  # fail if variable is undefined
+set -o pipefail  # fail if command before pipe fails
+
+usage() {
+    echo "Usage: $0 -t <tenant-uuid> [-u <auth-username>]"
+    exit 1
+}
+
+# Check for dependencies
+if ! command -v wazo-auth-cli &> /dev/null; then
+    echo "wazo-auth-cli is required to execute this script"
+    exit 2
+fi
+
+if ! command -v wazo-confd-cli &> /dev/null; then
+    echo "wazo-confd-cli is required to execute this script"
+    exit 2
+fi
+
+# Parse command line arguments
+while getopts t:u: flag
+do
+    case "${flag}" in
+        t) TENANT_UUID="${OPTARG}";;
+        u) AUTH_USERNAME="${OPTARG}";;
+    esac
+done
+
+if [[ ! -v TENANT_UUID ]]; then
```

Comment:
This condition is also true if `$TENANT_UUID=''`. So I think `[ -z "$TENANT_UUID" ]` would be better.